sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -2539,13 +2539,13 @@ async function listEntities(
     case EntityType.QUEUE:
       const queueResponse = await serviceBusAtomManagementClient.getQueues({
         skip: skip,
-        top: top
+        maxCount: top
```

Comment:
why not rename the parameter name so this can be just `maxCount`?

Reply:
Can do.